src/components/Settings/sections/ModelManagementSection.scss

```
@@ -753,3 +753,33 @@
+  flex-direction: column;
+  gap: 8px;
+  padding: 12px;
+  border: 1px solid rgba(vars.$color-error, 0.3);
+  border-radius: 6px;
+
+  p {
+    margin: 0;
+    font-size: vars.$font-caption;
+    color: vars.$color-error;
+    line-height: 1.4;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)